### ISSUE TITLE
Stuck on Touch Screen Calibration (please click on the red dot) after flashing firmware

### ISSUE THREAD

**Issue by pdsavas** [bigtreetech/BTT-TFT35-E3-V3.0#42]
I have been trying to get a brand new screen working. I flashed the firmware, but when I get to the Touch Screen Calibration, the TFT is not responding to touch, so it sits at waiting for the red dot in the upper left corner of the screen to be pressed.

I'm fairly new to this so any help would be appreciated.

Thanks,
pete

**Comment by oldman4U** [comment]
Hi. Are you using pre or self compiled firmware. SD card size and format. Which files are on the card. Whats the name of the files before and after update?

**Comment by pdsavas** [comment]
I've tried using pre-compiled versions of v25 and v26. The SD card is a 1GB and formatted in FAT32.

The files on the card for v25 are: BIGTREE_TFT35_V3.0.25.bin (109,312 bytes both before and after update) before update and BIGTREE_TFT35>V3.0.25.CUR after update. Also, the entire TFT35 directory. The TFT35 directory gets renamed TFT35.CUR after the update.

The files for v26 are: BIGTREE_TFT35_V3.0.26.1.bin (162,584 bytes both before and after update) before update and BIGTREE_TFT35_V3.0.26.1.CUR after update. Also, the entire TFT35 directory. The TFT35 directory gets renamed TFT35.CUR after the update.

I even tried using the files from your fork (BIGTREETECH-TouchScreenFirmware-Oldman4U) with no success.

With all three, I used the files from the Unified Menu Material theme.

Please let me know if you need more information.

**Comment by oldman4U** [comment]
You have to use E3 based firmware version and update TFT35 folder as well
as config.ini like described on main page.
pdsavas <notifications@github.com> schrieb am Fr. 9. Okt. 2020 um 09:24:

**Comment by pdsavas** [comment]
When I tried using the E3 based firmare, I got a screen, after the update, that said:

Found outdated data:

Config: Update required

Insert the SD card with the required
files and press the reset button
to update.

**Comment by oldman4U** [comment]
Thats right. And the error is because you are not reading.

pdsavas <notifications@github.com> schrieb am Fr. 9. Okt. 2020 um 09:31:

**Comment by pdsavas** [comment]
Please then, instead of being condescending, tell me what I am missing. As I said, I'm VERY new to all of this and trying hard to figure it out. I've been reading and watching videos for weeks now, and yet I'm still missing something basic, apparently.

**Comment by oldman4U** [comment]
Your card is fine, using E3 firmware is correct and when you read the ReadMe and the error you have seen on screen you will also know what to do. 

Config: Update required

Insert the SD card with the required files and press the reset button to update.

**Comment by pdsavas** [comment]
Okay. I've gone back and re-read the pages (both the main and the config instructions). I THINK I have it figured out and am trying to flash the display once more (this time with a config.ini file on the card). I'm using the BIGTREE_TFT35>V3.0_E3.26.bin (224,960 bytes) from the Unified Menu Material folder of the Master build.

And...success with the TFT!

I apologize for being frustrated. I've actually been building this printer from scratch (after having an Ender 3 for a while) using an Ender Extender XXL kit as the base frame. I've had to buy all the parts, and make cables needed that didn't come with the kit, so it's been a two-month-long learning process for me; I haven't even been able to test the hardware to see if it's going to work properly before trying the hot end and finally printing. Getting the screen working was the last thing I needed to do before actually checking if the motors will work, if the BLTouch will work, etc. It's been nerve-wracking at best; frustrating at worst.

Thank you again.
pete
